### ISSUE TITLE
Blender.org changed the download page layout - breaking changes for the daily and experimental web scrapes.

### ISSUE THREAD

**Issue by JJeris** [DotBow/Blender-Launcher#241]
**Describe the bug**
As the title says, both the daily download and experimental download pages had their UI changed for...no good reason. Its more readable, but feels very disjointed from the rest of the page.

Anyways, this means that your webscrape is now broken, and Blender Launcher no longer is able to download daily or experimental builds.

https://builder.blender.org/download/daily/

p.s. the read me displays blender.org links, that no longer exist.

**To Reproduce**
Open Blender Launcher>Daily/Experimental>"No new builds available.".

**Expected behavior**
That there are builds available.

**System (please complete the following information):**
 - Windows 10
 - Blender Launcher version:1.15.1

**Additional context**
I only noticed this because im building a tool for Blender that also allows people to download Blenders.

**Comment by xan2622** [comment]
I can't believe I am saying this (as I am probably the number one fan of this project and I wish [its development is resumed](https://github.com/DotBow/Blender-Launcher/discussions/188)), but if this means that *Blender Launcher* no longer works, maybe that's a good thing for the official "Blender downloads servers". 

I am saying this regarding this other thread: 👉 https://github.com/DotBow/Blender-Launcher/issues/213

**Comment by JJeris** [comment]
Me and and a few others are building something similar to Blender Launcher, but we also used webscraping (for one, were still building and prototyping the app, and for two, Blender Launcher was using it, but i had no idea that BL was abandoned).

In the issue you showed, @xan2622, do they mean that the best option is for BL to host its own server with the Blender builds? Or, a DB?
What did they mean by service?

**Comment by xan2622** [comment]
You might want to ask official Blender representatives (or developers) at https://blender.chat 
Or ask *mcgrathd* in the #213 thread.
I am not qualified to give you the best technical answer for this question.

**Comment by thinsoldier** [comment]
> Me and and a few others are building something similar to Blender Launcher, but we also used webscraping (for one, were still building and prototyping the app, and for two, Blender Launcher was using it, but i had no idea that BL was abandoned).
> 
> In the issue you showed, @xan2622, do they mean that the best option is for BL to host its own server with the Blender builds? Or, a DB? What did they mean by service?

An easy solution is to put the web scraping side of things on a web server and have all of the desktop clients consult that pre-scraped and cached data instead of everyone doing all the same scraping over and over again wasting bandwidth and power. It also means a change to the web page can be fixed in the online scraper code instead of all users needing to download a new version of the desktop client just because a <div> was added to the web page.

**Comment by JJeris** [comment]
Yep, a web server would be what id go for too. Thanks guys

**Comment by Gerstmann-Bradley** [comment]
on blender chat they mentioned that they have a JSON end point and website scraping is no longer needed. for details you have to ask blender developers.

**Comment by JJeris** [comment]
Oh @Gerstmann-Bradley ? What Blender chat? That be exactly what im looking for.

**Comment by xan2622** [comment]
I shared the link a few posts above: https://github.com/DotBow/Blender-Launcher/issues/241#issuecomment-1816784625

**Comment by L0Lock** [comment]
https://twitter.com/tonroosendaal/status/1726565669590986932

Hopefully not, but could BL be the culprit? Or to a lesser extent a contributor of the DDOS? 👀

I disabled auto-updates a long time ago on mine and I hope as many ppl as possible did the same.

**Comment by JJeris** [comment]
@L0Lock def not. Ive read the BL code, and, though im not an expert on webscraping in python, if the webscraping fails, then it just fails. I dont believe it would try and scrape again, after failing to get any data, tho ofc you can set BL to do that every 10 mins.

If BL hadnt done this before, when it was more popular, i doubt it would now.

**Comment by Victor-IX** [comment]
You can download a fix version here:
https://github.com/Victor-IX/Blender-Launcher/releases/tag/v1.15.2

**Comment by vkleinmp** [comment]
> You can download a fix version here: https://github.com/Victor-IX/Blender-Launcher/releases/tag/v1.15.2

Stable builds are not shown, only Daily and only 2 in Experimental (there should be more)

**Comment by Victor-IX** [comment]
@vkleinmp Stable builds are not available for now on any version and I can't do anything about it, the web requests are currently blocked by the Blender Foundation.

**Comment by Gerstmann-Bradley** [comment]
again I would recommend to consult with developers on blender chat. lazydodo mentioned about json end point that makes web scrapping unnecessary.
while I don't understand anything, perhaps programmers can improve the entire method of Blender Launcher.

**Comment by Victor-IX** [comment]
We are already working on that 🙂! I already lowered the number of requests made to the website in the last version I made, but a more future-proof and optimized solution will be made soon

**Comment by JJeris** [comment]
If someone figures out more about the json end points, let me know.
